standard.mk

```diff
@@ -80,36 +68,31 @@ gocheck: ## Lint code
 	go vet ./cmd/... ./pkg/...
 
 .PHONY: gobuild
-gobuild: gocheck gotest
```

Comment:
Why remove lint and unit tests from build?

Reply:
They're run as separate steps, aren't they? We don't need to run them twice.

Reply:
Nope, they aren't (yet). Adding this back